serverless/installing_serverless/upgrading-serverless.adoc

```diff
@@ -0,0 +1,32 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="upgrading-serverless"]
+= Upgrading {ServerlessProductName}
+include::modules/common-attributes.adoc[]
+:context: upgrading-serverless
+
+toc::[]
+
+If you have previously installed a Technology Preview version of {ServerlessProductName}, follow the instructions in this guide to upgrade to the latest Generally Available version.
+
+In Technology Preview versions of {ServerlessProductName} (versions 1.6.0 and older), the only available Subscription Update Channel was `techpreview`.
```

Comment:
We're currently revisiting this as we might want to start rolling out to a similar-to-GA model even in 1.6.0. See https://github.com/openshift-knative/serverless-operator/pull/197 for the engineering PR. We'd release 1.6.0 only into the `preview-4.3` channel so we'd put these docs to a test even in 1.6.0.

Mostly leaving this here as a FYI that we might still want to do this for 1.6.0. Let's confirm in Slack once all the naming is confirmed.